code/modules/events/rogue_drones.dm

```diff
@@ -23,20 +23,21 @@
 	var/msg
 	var/rng = rand(1,5)
 	//VOREStation Edit Start TFF 16/12/19 - Sif -> Virgo 3b
```

Comment:
Feel free to remove this comment portion since you're making your own edit to the event.